handyrl/train.py

```diff
@@ -288,16 +296,18 @@ def select_episode(self):
             if random.random() < accept_rate:
                 break
         ep = self.episodes[ep_idx]
-        turn_candidates = 1 + max(0, ep['steps'] - self.args['forward_steps'])  # change start turn by sequence length
-        st = random.randrange(turn_candidates)
-        ed = min(st + self.args['forward_steps'], ep['steps'])
+        trained_steps = self.args['forward_steps'] - self.args['burn_in_steps']
+        turn_candidates = 1 + max(0, ep['steps'] - trained_steps)  # change start turn by sequence length
+        st_train = random.randrange(turn_candidates)
+        ed = min(st_train + trained_steps, ep['steps'])
+        st = max(0, st_train - self.args['burn_in_steps'])
```

Comment:
I wonder which of the following two is better.

When forward_steps=24 and burn_in_steps=8, 
1. use first 8 samples of 24 samples in batch for burn-in (i.e., training with 16 samples)
2. use 8 samples for burn-in and 24 samples for training respectively  (i.e. batch size is 32)
    3. can this separate the GPU calculation between burn-in and training in `forward_prediction`?

At first, I have been thought it is the second behavior.

Reply:
Yes, feautre/burn_in_step_naive_forward_steps is the correct branch!

https://github.com/YuriCat/HandyRL/compare/feature/burn_in_steps...feautre/burn_in_step_naive_forward_steps